src/quo/components/gradient/gradient_cover/view.cljs

```diff
@@ -6,14 +6,21 @@
     [react-native.linear-gradient :as linear-gradient]))
 
 (defn- view-internal
-  [{:keys [customization-color opacity container-style height] :or {customization-color :blue}}]
-  (let [color-top    (colors/custom-color customization-color 50 20)
-        color-bottom (colors/custom-color customization-color 50 0)]
-    [linear-gradient/linear-gradient
-     {:accessibility-label :gradient-cover
-      :colors              [color-top color-bottom]
-      :start               {:x 0 :y 0}
-      :end                 {:x 0 :y 1}
-      :style               (merge (style/root-container opacity height) container-style)}]))
+  [{:keys [customization-color opacity container-style height]
+    :or   {customization-color :blue}}]
+  ;; `when` added for safety, `linear-gradient` will break if `nil` is passed,
```

Comment:
why don't we just wrap the linear gradient in a `when` to do a nil check. Or better again do this directly in `react-native.linear-gradient`. We can then remove this comment as it is a generic solution 👍

Reply:
I think it's better solve in separate issue